python/3.py

```diff
@@ -0,0 +1,36 @@
+import math
+
+limit = 600851475143
+max_factor = int(math.floor(math.sqrt(limit)))
+
+
+def is_prime(n):
+    if n < 2:
+        return False
+    elif n == 2:
+        return True
+    else:
+        for i in range(3, int(math.sqrt(n)) + 1, 2):
+            if n % i == 0:
+                return False
+
+    return True
+
+factors = []
+for n in range(2, max_factor):
+    if limit % n == 0:
+        factors.append(n)
+        factors.append(limit / n)
+
+factors = sorted(factors, key=lambda x: -x)
```

Comment:
because `is_prime` is extra computation i think reversed will be less processing, no math proof from me tho.